common/views/components/WorkMedia/WorkMedia2.js

```diff
@@ -0,0 +1,76 @@
+// @flow
+import {grid, font, spacing} from '../../../utils/classnames';
+import Icon from '../Icon/Icon';
+import ImageViewer2 from '../ImageViewer/ImageViewer2';
+import ScrollToInfo from '../ScrollToInfo/ScrollToInfo';
+import {iiifImageTemplate} from '../../../utils/convert-image-uri';
+
+type Props = {|
+  id: string,
+  title: string,
+  iiifUrl: string,
+  width?: number,
+  queryString?: string,
+|}
+
+const href = (queryString, id) => {
```

Comment:
Why the abstraction?

The Image / Viewer etc components have quite a lot of 1-off abstrations, making it a little hard to read the flow of the Component (for me).

Reply:
This wasn't actually part of this PR, in the sense that I just copied the WorkMedia.js file, so I could use ImageViewer2.js in it instead - so the current apps still worked.

Reply:
But isn't the point of V2 to be a refactor - not evolution?